openstack/dns/v2/recordsets/results.go

```diff
@@ -111,16 +115,20 @@ type RecordSet struct {
 	// Links includes HTTP references to the itself,
 	// useful for passing along to other APIs that might want a recordset
 	// reference.
-	Links []golangsdk.Link `json:"-"`
+	Links *LinkResult `json:"links"`
```

Comment:
please revert this, we need to reanalyze this.

Reply:
Done.